tests/test_cli.py

```diff
@@ -198,3 +229,36 @@ def test_render_documents(cli_mocks: CLIMocks):
     assert (
         cli_mocks.update_documents.call_args_list[1].args[1] == "TestProject"
     )
+
+
+def test_run_plugins(cli_mocks: CLIMocks):
+    command: list[str] = [
+        *GROUP_COMMAND,
+        "run-plugins",
+        "--document-rendering-config",
+        str(TEST_COMBINED_DOCUMENT_CONFIG),
+        "--synchronize-config",
+        str(TEST_MODEL_ELEMENTS_CONFIG),
+        "--grouped-links-custom-fields",
+        "--plugin-config-file",
+        str(TEST_PLUGIN_CONFIG),
+    ]
+
+    result = testing.CliRunner().invoke(main.cli, command, terminal_width=60)
+
+    assert result.exit_code == 0
+    assert len(cli_mocks.dummy_plugin_1.method_calls) == 4
+    assert cli_mocks.dummy_plugin_1.init.call_count == 2
+    assert len(cli_mocks.dummy_plugin_1.init.call_args_list[0].args) == 3
+    assert len(cli_mocks.dummy_plugin_1.init.call_args_list[1].args) == 3
+    assert cli_mocks.dummy_plugin_1.init.call_args_list[0].kwargs == {"a": 12}
+    assert cli_mocks.dummy_plugin_1.init.call_args_list[1].kwargs == {"a": 98}
+    assert cli_mocks.dummy_plugin_1.run.call_count == 2
+    assert cli_mocks.dummy_plugin_1.run.call_args_list[0].kwargs == {
+        "b": "abc"
+    }
+    assert cli_mocks.dummy_plugin_1.run.call_args_list[1].kwargs == {
+        "b": "xyz"
+    }
+
+    assert len(cli_mocks.dummy_plugin_2.method_calls) == 0
```

Comment:
Plugin 2 is not configured that's why there is no method call I suppose? Then everything is fine.